subprojects/soak/src/integTest/groovy/org/gradle/launcher/daemon/DaemonPerformanceMonitoringSoakTest.groovy

```diff
@@ -35,25 +35,39 @@ class DaemonPerformanceMonitoringSoakTest extends DaemonMultiJdkIntegrationTest
         )
     }
 
-    def "when build leaks slowly daemon is eventually expired"() {
+    @Unroll
+    def "when build leaks heap space slowly daemon is eventually expired (maxBuilds=#maxBuilds, heapSize=#heapSize, leakRate=#leakRate"() {
         when:
         setupTenuredHeapLeak(leakRate)
+
         then:
-        daemonIsExpiredEagerly(maxBuilds, heapSize)
+        daemonIsExpiredEagerly(maxBuilds, heapSize, heapSize * 2)
 
         where:
         maxBuilds | heapSize | leakRate
-        45        | "200m"   | 600
```

Comment:
The original rates were not enough to get to our heap usage threshold. Not sure how this passed previously. Must have been some time ago with different thresholds?